spark_llm/prompt.py

```diff
@@ -119,3 +119,64 @@
 Assume the result of the Spark SQL query is stored in a dataframe named 'df', visualize the query result using plotly.
 There is no need to install any package with pip.
 """
+
+VERIFY_TEMPLATE = """
+Given 1) a PySpark dataframe, my_df, and 2) a description of expected properties, desc,
+generate a Python function to test whether the given dataframe satisfies the expected properties.
+Your generated function should take 1 parameter, df, and the return type should be a boolean.
+You will call the function, passing in my_df as the parameter, and return the output (True/False).
```

Comment:
shall we ask the llm to generate the code that contains the function, and we can execute the generated code to get the result (True/False). I don't think the llm can directly execute the code generated.

Reply:
yep, fixed!